lib/src/tests/mod.rs

```diff
@@ -94,132 +101,165 @@ fn tast_to_ast(class: &Class) -> Class {
 
-    // Write AST & TAST to files
-    let mut file =
-        File::create(format!("tests/{name}-AST.json")).expect("File couldn't be created");
-    serde_json::to_writer_pretty(&mut file, &ast).expect("Couldn't serialize class");
-
-    if let Some(tast) = tast {
-        let mut file =
-            File::create(format!("tests/{name}-TAST.json")).expect("File couldn't be created");
-        serde_json::to_writer_pretty(&mut file, tast).expect("Couldn't serialize class");
-    }
+    assert_eq!(og_clz_file, gen_clz_file);
+    Ok(())
 }
 
 // use super::*\n#[test]
 // fn Fields_class() {
 //     let class = Class {};
-//     create_test_file(&tast_to_ast(&class), Some(&class), "Fields");
+//     single_class_test(&tast_to_ast(&class), Some(&class), "Fields");
 // }
 
 // use super::*\n#[test]
 // fn IntFields_class() {
 //     let class = Class {};
-//     create_test_file(&tast_to_ast(&class), Some(&class), "IntFields");
+//     single_class_test(&tast_to_ast(&class), Some(&class), "IntFields");
 // }
 
 // use super::*\n#[test]
 // fn LocalVarDecl_class() {
 //     let class = Class {};
-//     create_test_file(&tast_to_ast(&class), Some(&class), "LocalVarDecl");
+//     single_class_test(&tast_to_ast(&class), Some(&class), "LocalVarDecl");
 // }
 
 // use super::*\n#[test]
 // fn MethodCall_class() {
 //     let class = Class {};
-//     create_test_file(&tast_to_ast(&class), Some(&class), "MethodCall");
+//     single_class_test(&tast_to_ast(&class), Some(&class), "MethodCall");
 // }
 
 // use super::*\n#[test]
 // fn NamingConflict_class() {
 //     let class = Class {};
-//     create_test_file(&tast_to_ast(&class), Some(&class), "NamingConflict");
+//     single_class_test(&tast_to_ast(&class), Some(&class), "NamingConflict");
 // }
 
 // use super::*\n#[test]
 // fn Negator_class() {
 //     let class = Class {};
-//     create_test_file(&tast_to_ast(&class), Some(&class), "Negator");
+//     single_class_test(&tast_to_ast(&class), Some(&class), "Negator");
 // }
 
 // use super::*\n#[test]
 // fn Return_class() {
 //     let class = Class {};
-//     create_test_file(&tast_to_ast(&class), Some(&class), "Return");
+//     single_class_test(&tast_to_ast(&class), Some(&class), "Return");
 // }
 
 // use super::*\n#[test]
 // fn SetterGetter_class() {
 //     let class = Class {};
-//     create_test_file(&tast_to_ast(&class), Some(&class), "SetterGetter");
+//     single_class_test(&tast_to_ast(&class), Some(&class), "SetterGetter");
 // }
 
 // use super::*\n#[test]
 // fn StrAdd_class() {
 //     let class = Class {};
-//     create_test_file(&tast_to_ast(&class), Some(&class), "StrAdd");
+//     single_class_test(&tast_to_ast(&class), Some(&class), "StrAdd");
 // }
 
 // use super::*\n#[test]
 // fn While_class() {
 //     let class = Class {};
-//     create_test_file(&tast_to_ast(&class), Some(&class), "While");
+//     single_class_test(&tast_to_ast(&class), Some(&class), "While");
 // }
```

Comment:
Im guessing this will be enabled later on, otherwise it's probably better to delete it for readability

Reply:
Yes, I will enable them, once I get around to writing the expected TASTs for them by hand